utils_nlp/eval/evaluate_summarization.py

```diff
@@ -0,0 +1,33 @@
+import os
```

Comment:
Can we shorten this module name to summarization? Since it's already under the eval folder. I recently shorten "evaluate_question_answering.py" to "question_answering.py" because it was too long.